realm/realm-library/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -62,7 +62,7 @@ public DynamicRealmObject(RealmObject obj) {
 
     DynamicRealmObject(BaseRealm realm, Row row) {
         this.realm = realm;
-        this.row = (row instanceof CheckedRow) ? (CheckedRow) row : ((UncheckedRow) row).convertToChecked();
+        this.row = row instanceof CheckedRow ? (CheckedRow) row : ((UncheckedRow) row).convertToChecked();
```

Comment:
I'm with @cmelchior. In certain instances I also find additional parens increase readability. Readability > minimalist code. The less cognitive load a developer has to perform the less likely they are to make an error.